assets/js/blocks/product-collection/inspector-controls/created-control.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Flex,
+	FlexItem,
+	RadioControl,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalToggleGroupControl` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalToggleGroupControlOption` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	// @ts-expect-error Using experimental features
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { ETimeFrameOperator, QueryControlProps } from '../types';
+
+const CreatedControl = ( props: QueryControlProps ) => {
+	const { query, setQueryAttribute } = props;
+	const { timeFrame } = query;
+
+	return (
+		<ToolsPanelItem
+			label={ __( 'Created', 'woo-gutenberg-products-block' ) }
+			hasValue={ () => timeFrame?.operator && timeFrame?.value }
+			onDeselect={ () => {
+				setQueryAttribute( {
+					timeFrame: undefined,
+				} );
+			} }
+		>
+			<Flex direction="column" gap={ 3 }>
+				<FlexItem>
+					<ToggleGroupControl
+						label={ __(
+							'Created',
+							'woo-gutenberg-products-block'
+						) }
+						isBlock
+						onChange={ ( value: ETimeFrameOperator ) => {
+							setQueryAttribute( {
+								timeFrame: {
+									...timeFrame,
+									operator: value,
+								},
+							} );
+						} }
+						value={ timeFrame?.operator || ETimeFrameOperator.IN }
+					>
+						<ToggleGroupControlOption
+							value={ ETimeFrameOperator.IN }
+							label={ __( 'IN', 'woo-gutenberg-products-block' ) }
+						/>
+						<ToggleGroupControlOption
+							value={ ETimeFrameOperator.NOT_IN }
+							label={ __(
+								'Not in',
+								'woo-gutenberg-products-block'
+							) }
+						/>
```

Comment:
Nit: is it intentional that `IN` is uppercase while `Not in` isn't?

![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/88ecc9cd-eba5-4c2e-b99d-23bfe038c327)

Reply:
@Aljullu, I've referenced the design from the provided screenshot; however, I'm unable to locate it in Figma. @kmanijak, could you please share the Figma design's URL? 🙂

To ensure consistency, we could:
- Capitalize all letters: IN & NOT IN
- Or capitalize only the first letter: In & Not in

@manospsyx What do you think? 🙂

Reply:
I'd say to use capitalized letters since the "In" might not look great, but I'd wait for Mano's input! 

Another nit: It would be better to use the `_x()` function with context instead of the generic `__()`. These words are small, making it difficult to understand what is actually being translated.

Reply:
I'd probably:

- capitalize both (preferably via CSS; it's not great to rely on localizable strings for styling), and 
- use a slightly smaller font size, perhaps by 5-10%, or 1-2px :)

Reply:
@xristos3490 

> Another nit: It would be better to use the _x() function with context instead of the generic __(). These words are small, making it difficult to understand what is actually being translated.

Makes sense to me. What do you think about the following contexts?

```tsx
_x('Not in', 'Exclude products created within a certain time frame', 'woo-gutenberg-products-block');

_x('In', 'Include products created within a certain time frame', 'woo-gutenberg-products-block');
```

I can make the changes once we finalize the context text.

Reply:
@manospsyx 
> capitalize both (preferably via CSS; it's not great to rely on localizable strings for styling), and

Sounds good 👍 

> use a slightly smaller font size, perhaps by 5-10%, or 1-2px :)

I am utilizing the [ToggleGroupControl](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-togglegroupcontrol--docs) component from @wordpress/components. Given this, I am hesitant to manually adjust the font size with custom CSS since the component currently offers only two size variants: "default" and "__unstable-large." It may be prudent to retain the current sizing, and revisit this matter once more options are made available within the ToggleGroupControl. It's important to note that any future modifications should also be applied to the Layout toggle to maintain consistency between the two controls.

<img width="256" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/41e4c9a7-9493-4ecf-af48-01ab20f885b4">

What do you think? 🙂

Reply:
Good point, @imanish003 -- it's best to stick with the design of the component itself for consistency. In this case, let's go with capitalizing the first letter of the first word. 

On a related note and regarding the text string itself, I would use "In" / "Not In" to refer to lists of items = when the value is a multi-select input. 

In this case, I think it's more accurate/clear to go with:

- Within
- Before

Reply:
> let's go with capitalizing the first letter of the first word.

Sounds good to me 🙌🏻

> In this case, I think it's more accurate/clear to go with:
Within
Before

Makes sense. I will make the changes.

Reply:
> Makes sense to me. What do you think about the following contexts?

I would keep this strictly to the broader context. Like:
```js
_x('Not in', 'Product Collection query operator', 'woo-gutenberg-products-block');
_x('In', 'Product Collection query operator', 'woo-gutenberg-products-block');
```

Reply:
@manospsyx I have made the changes & now UI looks like this:
<img width="281" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/c2623af7-1d19-428b-a7e5-0fe034374b08">

Reply:
@xristos3490 Okay, I will go ahead & will make the changes. Thanks 🙏🏻

Reply:
> That said, It's also advisable to provide context for other instances here like "Stack" or "Grid".

Makes sense. What do you think about the following?
```
_x('Stack', 'layout style for displaying products in a single column', 'woo-gutenberg-products-block');
_x('Grid', 'layout style for displaying products in multiple columns', 'woo-gutenberg-products-block');
```

Reply:
In my opinion, providing an exact description of the context could result in excessive wordiness. A simple term such as "Product Collection query operator" should suffice for translators to comprehend the intended meaning. However, I do not have a strong preference, so I leave it up to you to decide. :)

Reply:
I don't have a strong opinion either. I will go ahead and add "Product Collection query operator". Thanks 🙏🏻